demos/projects/ESPRESSIF/aduotaupdate/port/azure_iot_flash_platform.c

```diff
@@ -0,0 +1,26 @@
+
+#include "azure_iot_flash_platform.h"
+
+AzureIoTResult_t AzureIoTPlatform_WriteBlock( AzureADUImage_t * const pxFileContext,
```

Comment:
So... What do you think about adding a couple more functions?
AzureIoTPlatform_OpenFile
> Tells the platform to prep the banks, plus how much data is supposed to be written through  _WriteBlock()
AzureIoTPlatform_CloseFile.
> Gives a flag indicating success or failure (in case anything goes wrong with the download.

Reply:
Yea I don't know enough about what is going to be needed in this case. I'd be okay with creating it as we need it.

Reply:
Fine. Let's get there first, I agree.

Reply:
It would be good to introduce it early. But np, it can be added next.